projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/side-bar.component.ts

```diff
@@ -138,29 +51,8 @@ export class SideBarComponent implements OnInit, OnChanges, OnDestroy {
     this.selectedView = sidebarView;
   }
 
-  onClick(sidebarView: string) {
-    this.toggleSidebarView(sidebarView);
-    this.resetEditBookmark();
-
-    const pdfLocation: PdfLocation = this.pdfWrapperProvider.pdfWrapper().getLocation();
-    this.addBookmark({
-      name: 'new bookmark',
-      pageNumber: pdfLocation.pageNumber - 1,
-      xCoordinate: pdfLocation.left,
-      yCoordinate: pdfLocation.top
-    } as Bookmark);
-  }
-
-  private extractDocumentId(url: string): string {
-    url = url.includes('/documents/') ? url.split('/documents/')[1] : url;
-    return url.replace('/binary', '');
+  onAddBookmarkClick() {
+    this.toggleSidebarView('bookmark');
+    this.store.dispatch(new CreateBookmark('new bookmark'));
```

Comment:
whats 'new bookmark'

Reply:
it's the text of the new bookmark. it's like a placeholder when they click the bookmark button, and then they can change that it